apps/wallet-mobile/src/components/ConfirmTx/ConfirmTx.tsx

```diff
@@ -317,7 +317,6 @@ const useStyles = () => {
   const {color} = useTheme()
   const styles = StyleSheet.create({
     root: {
-      flex: 1,
```

Comment:
@banklesss you added this recently, but I don't know where it was fixing something. In both places I saw it, it moved the button below the parent container outside the screen.

Reply:
sorry don't remember 👀

if your change works its ok 👍🏻